ipfsconn/ipfshttp/ipfshttp.go

```diff
@@ -496,12 +181,8 @@ func (ipfs *Connector) Shutdown() error {
 		return nil
 	}
 
-	logger.Info("stopping IPFS Proxy")
```

Comment:
would be good to log `stopping IPFS Connector`.

Reply:
Had missed this one.